scripts/soltest.sh

```diff
@@ -1,12 +1,32 @@
 #!/usr/bin/env bash
-
 set -e
 
 REPO_ROOT="$(dirname "$0")"/..
 USE_DEBUGGER=0
 DEBUGGER="gdb --args"
 BOOST_OPTIONS=
 SOLTEST_OPTIONS=
+BUILD_DIR=${BUILD_DIR:-build}
+
+usage() {
+    echo 2>&1 "
+Usage: $0 [options] [soltest-options]
+Runs BOOST C++ unit test program, soltest.
+
+Options:
+  --debug                  soltest invocation prefaced with: \"$DEBUGGER\"
+  --debugger *dbg-cmd*     soltest prefaced with your own debugger command.
+  --run_test | -t  *name*  filters test unit(s) to include or exclude from test.
+                           This  option can be given several times.
+  --boost-options *x*      Set BOOST option *x*.
+  --show-progress | -p     Set BOOST option --show-progress.
```

Comment:
Isn't the Boost option called `--show_progress`?

Reply:
*looks at command line*
`build/test/soltest --show-progress`

.. nope, it's correct like this

Reply:
That's a little bit confusing. Why does https://github.com/ethereum/solidity/pull/6462/files#diff-33c6b4e83faefd142a5bcda932de6863R86 say `show_progress` then?